packages/xo-server/src/xapi/index.js

```diff
@@ -1437,7 +1437,25 @@ export default class Xapi extends XapiBase {
     return /* await */ this._snapshotVm(this.getObject(vmId), nameLabel)
   }
 
-  async _startVm(vm, host, force) {
+  async _startVm(vm, host, force = false) {
+    if (!force) {
```

Comment:
The issue with using `force` for both condition is that the user won't be aware if they try to start a VM that has duplicated MAC address and for which start operation is blocked.